hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/MiniOzoneHAClusterImpl.java

```diff
@@ -408,10 +432,146 @@ protected void initOMRatisConf() {
       return omList;
     }
 
+    /**
+     * Start OM service with multiple OMs.
+     */
+    protected List<StorageContainerManager> createSCMService() throws IOException,
+        AuthenticationException {
+
+      List<StorageContainerManager> scmList = Lists.newArrayList();
+
+      int retryCount = 0;
+      int basePort = 12000;
+
+      while (true) {
+        try {
+          basePort = 12000 + RANDOM.nextInt(1000) * 4;
+          initSCMHAConfig(basePort);
+
+          for (int i = 1; i<= numOfSCMs; i++) {
+            // Set nodeId
+            String nodeId = SCM_NODE_ID_PREFIX + i;
+            String metaDirPath = path + "/" + nodeId;
+            OzoneConfiguration scmConfig = new OzoneConfiguration(conf);
+            scmConfig.set(OZONE_METADATA_DIRS, metaDirPath);
+            scmConfig.set(ScmConfigKeys.OZONE_SCM_NODE_ID_KEY, nodeId);
+            scmConfig.set(ScmConfigKeys.OZONE_SCM_HTTP_ADDRESS_KEY, "");
+            scmConfig.set(ScmConfigKeys.OZONE_SCM_HTTPS_ADDRESS_KEY, "");
+
+            // TODO: set SCM HA configs
+
+            configureSCM();
+            SCMStorageConfig scmStore = new SCMStorageConfig(scmConfig);
```

Comment:
Probably, its better to call ScmInit() instead of directly initialising the SCMStorage here. Similarly, for the other SCMs , we need to call scmBootStrap() once bootstrap is done.